presto-orc/src/test/java/com/facebook/presto/orc/TestFlatMap.java

```diff
@@ -355,6 +355,18 @@ public void testMixedEncodings()
                 ExpectedValuesBuilder.get(Function.identity()).setMixedEncodings());
     }
 
+    @Test
+    public void testIntegerWithMissingSequences()
+            throws Exception
+    {
+        // A test case where the sequences IDs for the ColumnEncodings for a map's keys aren't an entire range [0,N)
```

Comment:
nit: is it actually in range (1, N] ? Since additional sequence id always starts from 1

Reply:
I was including 0 since it's always present, it's just not and "additional" sequence ID.  I can see why it's confusing though, I'll clarify my comment.